### DISCUSSION TITLE
A couple of questions about note naming and dates formatting

### DISCUSSION THREAD

**Discussion by TigBuh99** [marcusolsson/obsidian-projects#431] [Help] [answered]
Please forgive me if the answers to my questions are obvious, but I've only started using the Projects plugin and I'm trying to understand it. I have the following questions:

1. If I want to name my new notes for a project so that they start with YYYY-MM-DD HH-mm and then some text, how may I do this? I see the setting for the default name for new notes in the Edit project window where it mentions supporting {{Date} and {{Time}}, but I can't make it work. Assume that I'm trying to use a template for new notes.
2. If I enter a "date:" field in YAML the date value is initially shown in Table view formatted as M/DD/YYYY. If I double click in the cell it changes to DD/MM/YYYY and the Calendar icon appears in the cell. After I click away it reverts to the initial format. How may I have the date shown as YYYY-MM-DD in the cell instead of M/DD/YYYY?
3. In the Configure Field window I cannot change the Type. is that by design?
4. How should I enter dates in YAML fields? With double square brackets or not?

**Comment by Acylation** [comment, marked as answer]
For the first question, the placeholders' keywords {{date}} and {{time}} are all in lowercase, and with formatting they can be written in {{date:YYYY-MM-DD-dddd}} and {{time:HH-mm}} styles, etc.

![image](https://user-images.githubusercontent.com/73122375/218678923-2bd123ce-f1a9-4c27-b0b7-ffbb507e0ced.png)

The second one sounds like a bug, as the date should be rendered according to your locale. If there is a strange behavior, would you mind adding a new issue to report it? Related to issue #153 .

For the third one, it's also unavailable for me now since the modal was updated.

Finally, only the YYYY-MM-DD format is supported now in the YAML front matter. Refer to discussion #309 .

**Reply by TigBuh99** [reply]
Thank you for your reply. Unfortunately I cannot get the title of new notes to show the date and time even though I changed the default name for new notes as per your illustration.
You are right about the second question, because my locale is USA. However, ideally one would want to be able to change this without having to change the locale which affects the system widely.
Your answers to my third and fourth question are noted.
